esp-hal-common/src/soc/esp32c3/efuse.rs

```diff
@@ -59,4 +59,25 @@ impl Efuse {
         let efuse = unsafe { &*EFUSE::ptr() };
         efuse.rd_repeat_data1.read().wdt_delay_sel().bits()
     }
+
+    /// Read Efuse data and return them
+    pub fn read_efuse(address: u32, offset: u32, size: u32) -> u32 {
```

Comment:
Efuses also can be read via peripheral access like so:
```rust
let efuse = unsafe { &*EFUSE::ptr() };

efuse.rd_sys_part1_data0.read().bit_field()
```
But seems currently bit fields isn't described.
(see registers definition in [esp32c3.base.svd](/esp-rs/esp-pacs/blob/4fe079193d4261319672bf91e4c11a5efb4a8268/esp32c3/svd/esp32c3.base.svd#L7542))